scanner.cs

```diff
@@ -1,8 +1,20 @@
 using System;
+using System.IO;
 
 public class Scanner {
-	public void initializeScanner(string file_name) {
-
+	private StreamReader __reader;
```

Comment:
We may want to try to read byte by byte, that way we can just count bytes read as our fp (easy to reset the fp to where we need to be, also). StreamReader is....meh. Unless you can figure out a way to reset the pointer.